frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/FinallyNode.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { chart_color_blue_300 as blueColor } from '@patternfly/react-tokens';
+import { observer, Node, Layer } from '@patternfly/react-topology';
+import { pipelineRunFilterReducer } from '../../../utils/pipeline-filter-reducer';
+import { PipelineVisualizationTask } from '../detail-page-tabs/pipeline-details/PipelineVisualizationTask';
+import {
+  BUILDER_NODE_ADD_RADIUS,
+  NODE_WIDTH,
+  NODE_HEIGHT,
+  FINALLY_NODE_PADDING,
+  FINALLY_NODE_VERTICAL_SPACING,
+  FINALLY_ADD_LINK_TEXT_HEIGHT,
+  BUILDER_NODE_ERROR_RADIUS,
+  FINALLY_ADD_LINK_SIZE,
+} from './const';
+import PlusNodeDecorator from './PlusNodeDecorator';
+import ErrorNodeDecorator from './ErrorNodeDecorator';
+import TaskList from './TaskList';
+
+const FinallyNode: React.FC<{ element: Node }> = ({ element }) => {
+  const {
+    clusterTaskList = [],
+    namespaceTaskList = [],
+    task,
+    pipeline,
+    pipelineRun,
+    disableBuilder,
+  } = element.getData();
+  const { t } = useTranslation();
+  const { finallyTasks = [], finallyListTasks = [] } = task;
+  const FINALLY_TASK_NODE_HEIGHT = NODE_HEIGHT + FINALLY_NODE_VERTICAL_SPACING;
+  const allTasksLength = finallyTasks.length + finallyListTasks.length;
+  return (
+    <Layer id="groups">
+      <>
+        <rect
+          className="odc-finally-node"
+          key={'rect'}
+          fill="transparent"
+          strokeWidth={1}
+          stroke="var(--pf-global--BorderColor--light-100)"
+          width={NODE_WIDTH + FINALLY_NODE_PADDING * 2}
+          height={
+            NODE_HEIGHT +
+            (allTasksLength - 1) * FINALLY_TASK_NODE_HEIGHT +
+            (!disableBuilder ? NODE_HEIGHT : 0) +
+            FINALLY_NODE_PADDING * 2
+          }
+          rx="20"
+          ry="20"
+        />
+
+        {finallyTasks.map((ft, i) => (
+          <g
+            key={ft.name}
+            className="odc-finally-builder-node"
+            transform={`translate(${FINALLY_NODE_PADDING}, ${FINALLY_TASK_NODE_HEIGHT * i +
+              FINALLY_NODE_PADDING})`}
```

Comment:
Suggest the same fully qualifying y
```suggestion
            transform={`translate(${FINALLY_NODE_PADDING}, ${NODE_HEIGHT * i + 
              FINALLY_NODE_VERTICAL_SPACE * (i - 1) + FINALLY_NODE_PADDING})`}
```
I understand it's more long winded, but it's easier to read imo.